lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -9,24 +9,27 @@ use Path::Class qw();
 class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
     is => 'Command::V2',
     has_input => [
+        model => {
+            is => 'Text',
+            doc => 'A RNASeq model that has ERCC transcripts spiked in.',
+            is_optional => '1,',
+        },
         bam_file => {
             is => 'FilePath',
-            doc => 'An aligned BAM file to a reference genome without the ERCC transcripts spiked in.',
+            doc => 'A path to a BAM file that has ERCC transcripts spiked in.',
+            is_optional => '1,',
         },
         ercc_fasta_file => {
             is => 'FilePath',
             doc => 'The FASTA format sequence for all ERCC transcripts.',
-            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa'],
+            default_value => '/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa',
+            is_optional => '1',
         },
         ercc_spike_in_file => {
             is => 'FilePath',
             doc => 'The control analysis file provided by Agilent for the ERCC spike-in.',
-            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC_Controls_Analysis.txt'],
-        },
-        ercc_spike_in_mix => {
-            is => 'Integer',
-            doc => 'The name of the Life Technologies ERCC spike-in mixture.',
-            valid_values => [ '1', '2'],
+            default_value => '/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC_Controls_Analysis.txt',
```

Comment:
After conversations with @apregier, @tmooney, and @nnutter , option 1 was finally taken.  See commit [89048b07b][1].

[1]: https://github.com/indraniel/genome/commit/89048b07bfc63d08f39a84d3c501bca9f3ddb901